showcase/src/assets/translations/fr.json

```diff
@@ -430,6 +430,49 @@
     "NEWS": {
       "TITLE": "Nouvelles"
     },
+    "NGX_FORM_ERRORS": {
+      "TITLE": "Reactive Forms errors",
+      "EXAMPLE": "Exemple",
+      "FORM": {
+        "VALIDATION": {
+          "REQUIRED": "{{fieldName}} est nécessaire",
+          "PASSWORD_REQUIRED": "{{fieldName}} doit être fourni",
+          "USER_NAME": {
+            "UNIQUE": "Votre nom d'utilisateur a déjà été pris"
+          },
+          "PASSWORD": {
+            "MAX_LENGTH": "Mot de passe ne peut pas y avoir plus de {{requiredLength}} caractères",
+            "MIN_LENGTH": "Mot de passe doit comporter au moins {{requiredLength}} caractères",
+            "PATTERN": "Votre mot de passe doit contenir au moins une majuscule, une minuscule et un chiffre."
+          },
+          "CONFIRM_PASSWORD": {
+            "ARE_EQUAL": "Non concordance des mots de passe passwords"
+          }
+        },
+        "HIDE_DETAILS": "Masquer les détails de validation",
+        "SHOW_DETAILS": "Afficher les détails de validation",
+        "HIDE_SUMMARY": "Masquer le résumé de validation",
+        "SHOW_SUMMARY": "Afficher le résumé de validation",
+        "SUBMIT": "Soumettre"
+      },
+      "FIELDS": {
+        "USER_NAME": "Nom d'utilisateur",
+        "PASSWORD": "Mot de passe",
+        "CONFIRM_PASSWORD": "Confirmez le mot de passe",
+        "ALIAS": {
+          "USER_NAME": "Votre nom d'utilisateur",
+          "PASSWORD_ALIAS": "Un code d'authentification valide",
+          "CONFIRM_PASSWORD": "Confirmation mot de passe"
+        },
+        "INFO": {
+          "HAS_ERRORS": "Contient des erreurs: {{hasErrors}}",
+          "HAS_SPECIFIC_ERROR": "Contient l'erreur '{{error}}': {{hasError}}",
+          "ERROR": "Erreur '{{error}}':",
+          "ERRORS": "Erreurs:",
+          "IS_TOUCHED": "Est touché: {{isTouched}}"
```

Comment:
Could you guys also have a look on these translations and tell me if they need to be corrected?